src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/ItemTooltip.java

```diff
@@ -25,43 +26,12 @@ public class ItemTooltip {
     private static volatile boolean sentNullWarning = false;
 
     /**
-     * Gets the NEU id from an id and an api id.
-     * @param id the id of the skyblock item, gotten from {@link de.hysky.skyblocker.utils.ItemUtils#getItemId(net.minecraft.item.ItemStack) ItemUtils#getItemId(ItemStack)} or {@link net.minecraft.item.ItemStack#getSkyblockId() ItemStack#getSkyblockId()}
-     * @param apiId the api id of the skyblock item, matching the id of the item on the Skyblocker api, gotten from {@link net.minecraft.item.ItemStack#getSkyblockApiId() ItemStack#getSkyblockApiId()}
-     * @return the NEU id of the skyblock item, matching the id of the item gotten from {@link io.github.moulberry.repo.data.NEUItem#getSkyblockItemId() NEUItem#getSkyblockItemId()} or {@link net.minecraft.item.ItemStack#getNeuName() ItemStack#getNeuName()},
-     * or an empty string if either id or apiId is null
+     * @deprecated Use {@link ItemUtils#getNeuId(ItemStack)} instead
      */
-    @NotNull
+    @Deprecated(since = "1.22")
```

Comment:
I would add a `.0` to the end so that it can't be mistaken for the next major minecraft version

Reply:
Done in 22fbe20.